Sources/SwiftFormat/Pipelines+Generated.swift

```diff
@@ -321,28 +321,28 @@ class LintPipeline: SyntaxVisitor {
 
 extension FormatPipeline {
 
-  func visit(_ node: Syntax) -> Syntax {
+  func rewrite(_ node: Syntax) -> Syntax {
```

Comment:
This file is generated by https://github.com/apple/swift-format/blob/main/Sources/generate-pipeline/PipelineGenerator.swift. Can you update that code and then rerun it to verify that the generated output here is correct?

Reply:
I totally overlooked that... Addressed and combine previous commits into one.